rollup.config.js

```diff
@@ -29,6 +29,37 @@ const umd_config = {
   ],
 };
 
+const cli_config = {
+  onwarn(message) {
+    if (message.code === "EVAL") { return; }
+    console.error(message);
+  },
+
+  input: "bin/peggy.mjs",
+  output: {
+    file   : "bin/peggy.js",
+    format : "cjs",
+    preferConst: true,
+    exports: "auto",
+    banner: "#!/usr/bin/env node",
```

Comment:
I think that adding a note about that this file is auto-generated also will be useful